pkg/knuu/instance.go

```diff
@@ -662,6 +662,11 @@ func (i *Instance) AddVolumeWithOwner(path, size string, owner int64) error {
 	if !i.IsInState(Preparing, Committed) {
 		return ErrAddingVolumeNotAllowed.WithParams(i.state.String())
 	}
+	// temporary feat, we will remove it once we can add multiple volumes
+	if len(i.volumes) > 0 {
```

Comment:
Would it make sense to add the check directly here?
https://github.com/celestiaorg/knuu/blob/1012d393b97b889a403ad6f5702b7bae50957277/pkg/knuu/instance.go#L654

Reply:
I'm sorry. You where right..

Reply:
I'm confused :D Friday mode..